lua/gitlab/actions/summary.lua

```diff
@@ -166,15 +167,16 @@ M.edit_summary = function()
 end
 
 M.create_layout = function(info_lines)
-  local title_popup = Popup(u.create_box_popup_state(nil, false))
+  local settings = u.merge(state.settings.popup, state.settings.popup.summary or {})
+  local title_popup = Popup(u.create_box_popup_state(nil, false, settings))
   M.title_bufnr = title_popup.bufnr
-  local description_popup = Popup(u.create_box_popup_state("Description", true))
```

Comment:
If we are splitting apart the popup functionality into it's own module that module should also contain `create_box_popup_state` and `create_popup_state`